### PR TITLE
Add the meetings page

### PR DESCRIPTION
**Pull request by jiachengzhang1**
Add the meetings page. Start the React app, and go to `http://localhost:3000/meetings` to see it.

### CODE REVIEW

**Comment by rsulli55** [comment]
Looks good to me!  Eslint is emitting some warnings, but I think those will be resolved down the line. @athanwalker anything you want to add?

File: frontend/src/Meetings.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Box, makeStyles, Typography } from "@material-ui/core";
```
Comment (rsulli55) [inline frontend/src/Meetings.js:1]: Could remove `makeStyles`

File: frontend/src/components/Meeting.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      <h2 id="meeting-title">Meeting</h2>
+      <h3 id="transition-modal-title">When is the meeting?</h3>
+      <p id="transition-modal-description">July 20, 2014</p>
+      <h3>What is the meeting level?</h3>
+      <p>Junior</p>
+      <h3>What are the topics?</h3>
+      <p>Arithmetic</p>
+      <h3>How can I join?</h3>
+      <p>
+        Join us on <a href="#">Zoom</a> and <a href="#">Miro</a>
```
Comment (rsulli55) [inline frontend/src/components/Meeting.js:23]: Im sure the warnings here will be fixed down the road.

**Review by rsulli55 — APPROVED** [review]

File: frontend/src/Meetings.js (hunk 1 above)
Comment (jiachengzhang1) [reply frontend/src/Meetings.js:1]: Removed

File: frontend/src/components/Meeting.js (hunk 2 above)
Comment (jiachengzhang1) [reply frontend/src/components/Meeting.js:23]: Yeah, many variables will be used when there are actual data

**Pull request merged**